sass/cd/cd-footer/_cd-soft-footer.scss

```diff
@@ -8,7 +8,7 @@
   padding: 24px 0;
   text-align: center;
   color: var(--cd-white);
-  background: rgba(var(--cd-rgb-ocha-blue), 0.9);
+  background: var(--brand-primary--light);
```

Comment:
I made this change because I'm not a huge fan of the variables defined in multiple formats. In all the other cases it was a pseudo-element whose opacity could substitute for an RGBA, so that `--brand-*` hex values could continue being used. Look at the main nav's active trail on an internal page as an example.

I might try the same pseudo-element approach here. Then we can retain both the simple DX of selecting a few hex colors, while retaining the automatic transparency on the soft footer.

@left23 what do you think?

Reply:
Yes, I know what you mean. IIRC there was some Designer thought gone into the degree of opacity in relation to the expected brand primary colour, to ensure text contrast.
I'm happy with ditching the RGBA. If the pseudo element doesn't work, the other options might be to use either `brand-primary--dark` or `brand-primary`. No strong opinions from me.
The soft footer is not widely used and when it is, I think it's customised in the sub theme.

Reply:
It looks like the colors aren't 1:1 when I switched to 90% of hex, but it is indeed nicer than the `--brand-primary--light` and the bonus is that it's still 100% automatic coloring 👍